tools/common-code/toolmanifest.pas

```diff
@@ -233,6 +238,23 @@   TCastleManifest = class
       that can optionally auto-create the source file. }
     property PluginSource: string read FPluginSource;
 
+    { Debian-specific things }
+    { Subfolder of /usr into which the app will be installed:
+      'games' will mean /usr/games/(projectname)
+      We make sure this directory will be set as a working directory when launched
+      Default: games }
+    property DebianInstallFolder: String read FDebianInstallFolder;
+    { App menu section, see man menufile for more information,
+      Default: Games }
+    property DebianSection: String read FDebianSection;
+    { Freedesktop category of the app,
+      see https://www.freedesktop.org/wiki/Specifications/menu-spec/ for more info
+      Default: Game }
+    property DebianCategories: String read FDebianCategories;
+    { A short description of the project.
+      See https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html for more info }
+    property DebianComment: String read FDebianComment;
```

Comment:
Same as above: call it `FreeDesktopComment`.

I know, it is also used as Debian package description later. That is OK -- we can always add new field specifically for multiline Debian description in the future. In the meantime, it makes sense we reuse desktop file short comment for it.

Reply:
Done in 2f1630dfc4257ef5d5879b52875dea04200dfc65